Wikipedia/Code/CIDetector+WMFFaceDetection.m

```diff
@@ -22,8 +22,8 @@ + (instancetype)wmf_sharedLowAccuracyBackgroundFaceDetector {
         defaultFaceDetector = [CIDetector detectorOfType:CIDetectorTypeFace
                                                  context:[CIContext wmf_sharedBackgroundContext]
                                                  options:@{
-                                   CIDetectorAccuracy: CIDetectorAccuracyLow,
-                                   CIDetectorMinFeatureSize: @(0.15)
+//                                   CIDetectorAccuracy: CIDetectorAccuracyLow,
```

Comment:
@bgerstle we don't pass anything if we want to get the defaults - why do you want to pass nil?